packages/client/ui/vanilla-ui/src/consts/version.ts

```diff
@@ -1 +1,3 @@
-export const LIB_VERSION = "1.1.15";
+import { version } from "../../package.json";
```

Comment:
Doing this includes package.json in the client bundle, which exposes the dependancies we use, and is problematic if we ship a package version w/ a vulnerable dep.

We may need to do something at build time instead here!

Reply:
> While the package.json content might be included in the source maps, for an open-source project this doesn't significantly alter the security landscape since the information is already public. However, if you're concerned about the source maps, you can configure your build process to exclude them from the production bundle or to generate source maps without including content from package.json.

Reply:
True, and since we include the version in the bundle it's not like the attacker couldn't trace back the version to the source code on npm or internet archives etc.